Document libgdiplus dependency (#15096)
dotnet/docs: docs/core/linux-prerequisites.md

* libunwind8
* libuuid1

> For .NET Core applications which use the System.Drawing.Common assembly, you will also need the following dependencies:

Reviewer: ```suggestion
For .NET Core applications that use the *System.Drawing.Common* assembly, you also need the following dependency:
```

> * libgdiplus (version 6.0.1 or later)

Reviewer: also wondering if we shouldn't just append this to the first list like this instead of creating a third list of dependencies

```suggestion
* libgdiplus (version 6.0.1 or later) - required by applications that use the *System.Drawing.Common* assembly.
```

> * libgdiplus (version 6.0.1 or later) - required by applications that use the *System.Drawing.Common* assembly.

Reviewer: That suggestion was an alternative version in case we wanted to combine this entry into the first list. If it's going to stay here, then we don't need to say it twice. Hopefully, it makes sense.

```suggestion
* libgdiplus (version 6.0.1 or later)
```

Author: Ah, makes sense 🤦‍♂ . Fixed it!

> [!NOTE]
> Most versions of Ubuntu include an earlier version of libgdiplus. You can install a recent version

> > of libgdiplus by adding the Mono repository to your system. See https://www.mono-project.com/download/stable/

Reviewer: ```suggestion
> of libgdiplus by adding the Mono repository to your system. For more information,
```

> > for more information.

Reviewer: ```suggestion
> see <https://www.mono-project.com/download/stable/>.
```

For more information about the dependencies, see [Self-contained Linux applications](https://github.com/dotnet/core/blob/master/Documentation/self-contained-linux-apps.md).

> For .NET Core applications which use the System.Drawing.Common assembly, you will also need the following dependencies:

Reviewer: ```suggestion
For .NET Core applications that use the *System.Drawing.Common* assembly, you'll also need the following dependency:
```

* libgdiplus (version 6.0.1 or later)

> [!NOTE]
> Most versions of CentOS and Fedora include an earlier version of libgdiplus. You can install a recent version

> > of libgdiplus by adding the Mono repository to your system. See https://www.mono-project.com/download/stable/

Reviewer: ```suggestion
> of libgdiplus by adding the Mono repository to your system. For more information,
```

> > for more information.

Reviewer: ```suggestion
> see <https://www.mono-project.com/download/stable/>.
```
